BranchSDK/BNCSKAdNetwork.m

```diff
@@ -183,7 +183,11 @@ - (BOOL) shouldCallPostbackForDataResponse:(NSDictionary *) dataResponseDictiona
     }
     
     int highestConversionValue = (int)[BNCPreferenceHelper sharedInstance].highestConversionValueSent;
-    if( conversionValue.intValue <= highestConversionValue ){
+    if((currentWindow == BranchSkanWindowFirst) && (conversionValue.intValue <= highestConversionValue)){
+        BOOL ascendingOnly = [self getAscendingOnlyFromDataResponse:dataResponseDictionary];
+        if (!ascendingOnly)
+            shouldCallUpdatePostback = YES;
+    } else if ((currentWindow != BranchSkanWindowFirst) && (highestConversionValue != 0) && (conversionValue.intValue <= highestConversionValue)){
```

Comment:
Is this second check just to prevent calling more than once with conversion value 0?

Reply:
@echo-branch  Now second and Third windows can have negative values (-1 (high), -2(medium), -3(low)) and default value for highestConversionValue (sent so for - used for ascendingOnly feature) is 0 for each window which is always higher than those negative conversion values. So in that scenario, if window is second or third and highest conversion value is 0 , it should go to last `else` condition and save that value in preferences.

Reply:
So the second check is cause we default `highestConversionValueSent` to 0, which would prevent negative conversion values from being saved. What if we default it to `NSIntegerMin` instead?